packages/manager/apps/web/client/app/domain/webhosting/order/domain-webhosting-order.routing.js

```diff
@@ -78,7 +77,9 @@ const resolve = {
         }),
       )
       .catch(() => {
-        displayErrorMessage('domain_webhosting_order_payment_checkout_error');
+        displayErrorMessage(
+          $translate.instant('domain_webhosting_order_payment_checkout_error'),
```

Comment:
If you are not providing an error message then the sentence will end with "panier :" right?
Maybe you should not modify the translation and simply add a new paragraph below with the error message

Reply:
Or maybe we should have 2 translations : first with error message (`domain_webhosting_order_payment_get_checkout_error`) and another one without (`domain_webhosting_order_payment_get_checkout_error_unknown`)

Reply:
@frenautvh not for that translation but for the other one. I'll update